ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -654,222 +654,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
-- as: aws-ipi-ovn-hypershift-guest-p1-f7
```

Comment:
Those are needed very soon. Instead of remove it, perhaps update the test frequency makes more sense ?

Reply:
This is one approach we discussed internally before. But based on current 4.12 and 4.13 debugging and HyperShift roadmap update, we may not just simplely copy all the 4.13 jobs to 4.14. We will review our profiles again in 4.14 and add some of jobs according to hypershift 4.14 planning.

Reply:
Make sense